### PR TITLE
Update to XAPI v24.14.0

### PR DESCRIPTION
**Pull request by benjamreis**

### CODE REVIEW

**Comment by benjamreis** [comment]
Waiting for dep build before bulding this one.

File: SPECS/xapi.spec [hunk 1]
```diff
@@ -65,18 +64,11 @@ Patch1002: xapi-23.31.0-open-openflow-port.XCP-ng.patch
 # Drop this patch when we don't want to support migration from older SDN controller anymore
 Patch1003: xapi-24.11.0-update-db-tunnel-protocol-from-other_config.XCP-ng.patch
 # Needed for IPv6 support. Upstream wants a better fix. Drop when upstream fixed it.
```
Comment (stormi) [inline SPECS/xapi.spec:66]: Is this comment still accurate?

**Review by stormi — APPROVED** [review]
Merging anyway so that I can resume the builds. You can check my previous comment afterwards and modify on master if needed.

**Comment by stormi** [comment]
Koji build to v8.3-incoming: http://koji.xcp-ng.org/taskinfo?taskID=57593

File: SPECS/xapi.spec (hunk 1 above)
Comment (benjamreis) [reply SPECS/xapi.spec:66]: no indeed, i'll fix it on master...

**Pull request merged**